code/modules/mining/equipment/survival_pod.dm

```diff
@@ -83,6 +83,11 @@
 	smooth = SMOOTH_MORE
 	canSmoothWith = list(/turf/closed/wall/mineral/titanium/survival, /obj/machinery/door/airlock/survival_pod, /obj/structure/window/shuttle/survival_pod)
 
+/obj/structure/window/reinforced/survival_pod
+	name = "pod window"
+	icon = 'icons/obj/lavaland/survival_pod.dmi'
+	icon_state = "pwindow"
```

Comment:
I don't believe windows handle diagonal directions properly, you might need to test that out for the like, one diagonal window corner you're adding here.